tests/integration/master_cluster/checks_clustercheck_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe file('/etc/xinetd.d/mysql_clustercheck.conf') do
+    it('should exist')
+    its('content') { should match /clustercheck password clustercheck 1 1/ }
```

Comment:
I was wrong here... (trap for @TomasHradecky) we should match "clustercheck clustercheck password" the user/password are switched